recipes/python-snap/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "python-snap" %}
+{% set version = "4.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://snap.stanford.edu/snappy/release/snap-4.0.0-4.0-centos6.5-x64-py2.6.tar.gz
+  sha256: dd621129151b49118b7952db14a6cc7be6d71ff9421f0d59751ab92d39ab3a47
+
+build:
+  number: 0
+  skip: True  # [not linux or not py27]
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - snap
+
+about:
+  home: http://snap.stanford.edu/snap/
+  license: BSD
+  license_file:
```

Comment:
If absent please comment this out and raise the issue upstream. (If you can add the issue link here it would be nice too.)